ta/trusted_keys/entry.c

```diff
@@ -87,11 +97,14 @@ static TEE_Result huk_ae_encrypt(TEE_OperationHandle crypto_op, uint8_t *in,
 {
 	TEE_Result res = TEE_ERROR_GENERIC;
 	struct tk_blob_hdr *hdr = (struct tk_blob_hdr *)out;
+	uint8_t iv[IV_SIZE] = { 0 };
 	uint32_t enc_key_len = in_sz;
 	uint32_t tag_len = TAG_SIZE;
 
 	hdr->reserved = 0;
-	TEE_GenerateRandom(hdr->iv, IV_SIZE);
+	TEE_GenerateRandom(iv, IV_SIZE);
+	memcpy(hdr->iv, iv, IV_SIZE);
+	memzero_explicit(iv, IV_SIZE);
```

Comment:
Why is this cleared?

Reply:
IV isn't considered sensitive, will drop it.